README.md

```diff
@@ -6,6 +6,43 @@
    pip install poetry && poetry install && poetry run pre-commit install
 ```
 
+## Trufflehog pre-commit hook
+
+- Installation on Mac
+
+```shell
+brew install trufflehog
+```
+
+- Installation via Docker
+
+Unix: 
+
+```shell
+docker run --rm -it -v "$PWD:/pwd" trufflesecurity/trufflehog:latest github --repo https://github.com/trufflesecurity/test_keys
+```
+
+M1 & M2 Mac
+
+```shell
+docker run --platform linux/arm64 --rm -it -v "$PWD:/pwd" trufflesecurity/trufflehog:latest github --repo https://github.com/trufflesecurity/test_keys
+```
+
+- Binary releases
+
+Download and unpack from https://github.com/trufflesecurity/trufflehog/releases
+
+- Once installed run ```poetry run pre-commit install``` to set up the git hook scripts 
+
+#### Running Trufflehog via Docker
+
+To run trufflehog in a Docker container there is a ```run-trufflehog.sh``` script which handles the Docker command and ensures your project directory is correctly mounted inside the container.
+
+Key Usage Note:
+
+For optimal hook efficacy, execute git add followed by git commit separately. This ensures TruffleHog analyzes all intended changes.
+Avoid using git commit -am, as it might bypass pre-commit hook execution for unstaged modifications.
```

Comment:
Do we want all the Docker instructions if the commit hook is just using the regular installed version?

Reply:
I agree with Will. Probably just remove all the alternative installation options and just point non-Mac users to the installation instructions trufflehog provides. If people don't want to install trufflehog they can work out the best way to use docker etc themselves.

Reply:
Agreed, I'll remove.